geoopt/manifolds/stiefel.py

```diff
@@ -138,20 +168,20 @@ def inner(self, x, u, v=None, *, keepdim=False):
             [-1, -2], keepdim=keepdim
         )
 
-    def _transp_follow_one(self, x, v, *, u):
+    def _transp_follow_one(self, x: torch.Tensor, v: torch.Tensor, *, u: torch.Tensor):
         a = self._amat(x, u)
         rhs = v + 1 / 2 * a @ v
         lhs = -1 / 2 * a
         lhs[..., torch.arange(a.shape[-2]), torch.arange(x.shape[-2])] += 1
         qv, _ = torch.solve(rhs, lhs)
         return qv
 
-    def transp_follow_retr(self, x, u, v):
+    def transp_follow_retr(self, x: torch.Tensor, u: torch.Tensor, v: torch.Tensor):
         return self._transp_follow_one(x, v, u=u)
 
     transp_follow_expmap = transp_follow_retr
 
-    def retr_transp(self, x, u, v):
+    def retr_transp(self, x: torch.Tensor, u: torch.Tensor, v: torch.Tensor):
         """Calculate an optimized retr_transp for Stiefel Manifold."""
         xvs = torch.cat((x, v), -1)
         qxvs = self._transp_follow_one(x, xvs, u=u).view(
```

Comment:
Not sure if we need `.view()` method anymore, since `.reshape()` is recommended with newer versions. But that is the minor note.

Reply:
I will fix that in the next PR